handlers.go

```diff
@@ -109,6 +110,12 @@ func (s *Server) ServeAPI(srv string, logging bool, lsteptok string, cachedir st
 		return nil
 	})
 
+	exporter := esmetrics.Exporter()
+	e.GET("/debug/metrics/opencensus", func(e echo.Context) error {
```

Comment:
@alvin-reyes @whyrusleeping is there a reason we aren't merging the opencensus and prometheus metrics together?

Reply:
I don't see why can't merge. I'm under the assumption that the `/prometheus` is the default now and we will migrate or switch over to `/opencensus. Once we have fully switched, we can remove the `/prometheus'